{"errorMessage": "'LOGGING_SERVER_NAME'", "errorType": "KeyError", "requestId": "0083dc4d-cfaf-4671-b07b-f11bd5e6f827", "stackTrace": ["  File \"/var/task/lambda_handler.py\", line 52, in handler\n    routes[target]()\n", "  File \"/var/task/lambda_handler.py\", line 41, in <lambda>\n    \"egresos_single\": lambda: run_egresos_single(\"egresos_detallado_pipeline\", _log_client()),\n", "  File \"/var/task/lambda_handler.py\", line 25, in _log_client\n    server_name=os.environ[\"LOGGING_SERVER_NAME\"],\n", "  File \"<frozen os>\", line 714, in __getitem__\n"]}